csrc/preseg_passes/move_pad.cpp

```diff
@@ -535,7 +535,8 @@ void replaceCat(Fusion* fusion) {
   // sanitizing CatOp with series of binary add
   for (auto* cat : ir_utils::filterByType<CatOp>(exprs)) {
     if (std::any_of(cat->inputs().begin(), cat->inputs().end(), [](Val* val) {
-          return !val->definition()->isA<PadOp>();
+          return val->definition() == nullptr ||
```

Comment:
Why is this?

Reply:
Yeah, you are right. I'm also not sure how this could have happened.

But I did just repro'ed it on b100. I'm not sure why this didn't show up on other arch.... I'll try to get a repro.

Reply:
Doesn't it happen because some of the cat inputs is just a size-0 tensor?

Reply:
If that happens it's a bug in remove_empty pass, since we should convert a CatOp to a set if there is only one non-empty input